.travis.yml

```diff
@@ -31,9 +31,8 @@ before_script:
 script:
   - go build -v ./cmd/dep
   - go vet $PKGS
-  # TODO (kris-nova) remove the ignore flag once we can refactor the code causing problems
   - staticcheck -ignore='github.com/golang/dep/cmd/dep/status.go:SA4010' $PKGS
-  #- ./hack/validate-gofmt.bash
+  - ./hack/validate-gofmt.bash
```

Comment:
Was this only temporarily disabled, or was codeclimate intended to permanently replace the fmt check?
I don't see a clear history, but for example the codeclimate change comment: "Remove govet and gofmt tests from travis which should speed things up." -@jessfraz